src/examples/java/io/nats/examples/ExampleUtils.java

```diff
@@ -132,4 +132,27 @@ private static void usage(String usageString) {
         System.out.println(usageString);
         System.exit(-1);
     }
+
+    public static String uniqueEnough() {
```

Comment:
Why not use a NUID?  `NUID.nextGlobal();`

Reply:
I specifically wanted a short, readable but unique enough piece of information. Nuid is almost 3 times as long as I wanted as well and is mixed case. Similar reason to not just using the currentTimeMillis